### ISSUE TITLE
Justification for B1 correction factor = 0.04

### ISSUE THREAD

**Issue by jcohenadad** [qMRLab/qMRLab#196]
[this hard-coded definition](https://github.com/neuropoly/qMRLab/blob/master/Models/Magnetization_transfer/mt_sat.m#L101) is based on [this reference](https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full).
What I find problematic is that there is no explanation about the origin of this number, and the article by Weiskopf et al. states that the "derivation and details of this correction will be reported elsewhere". Does anyone know if the "elsewhere" already exists somewhere?

**Comment by mathieuboudreau** [comment]
@jcohenadad I agree – I believe I asked @ileppe this question when we first implemented MTsat to qMRLab, but she didn't know either. 

Maybe we should email the authors? Did you already do so in the past Ilana?

Regardless, it might be important to make this correction factor an optional GUI element, as I think it has only been used/calculated/estimated for 3T.

**Comment by ileppe** [comment]
A while back, Nikola spoke to Gunther Helms about this, and he said he had "calibrated a phenomenological model" which is implemented in the MPM toolbox. I hadn't found it at the time but  I now see this http://kolbrek.hoyttalerdesign.no/index.php/sw/mpm-toolbox. I can investigate further if you like.

**Comment by ileppe** [comment]
@jcohenadad we found a potential repository (seemingly un-touched since 2012), which I think is from that group, where they hard-code the value, but it does not seem to be justified there
https://github.com/sadaszewski/vbq/blob/master/MTProt.m#L336

**Comment by agahkarakuzu** [comment]
Closed: #317. Discussion can be useful for users, to be moved to forum.

**Comment by helms-man** [comment]
can someone open #196 again?
here's my 2015 ismrm abstract about the b1+ correction of MTsat. planning to finally submit this. /gunther
[Helms-ProceedingsISMRM-23(2015)-3360.pdf](https://github.com/qMRLab/qMRLab/files/3292967/Helms-ProceedingsISMRM-23.2015.-3360.pdf)

**Comment by helms-man** [comment]
just an important remark on the abstract: the correction factor 0.4 has been empirically determined for one particular MPM protocol featuring a gaussian 4ms MT-pulse at 2kHz offset and 220 deg nominal flip angle. the calibration method, however, is generally applicable. hope this is of some help.

**Comment by mathieuboudreau** [comment]
This is all very useful information - thank you for clarifying this issue @helms-man !
